scanhub-ui/src/components/TaskCreateModal.tsx

```diff
@@ -25,20 +24,25 @@ import { useMutation } from 'react-query'
 
 import LoginContext from '../LoginContext'
 import { taskApi } from '../api'
-import { BaseTask, TaskOut, TaskType } from '../generated-client/exam'
-import { ModalComponentProps } from '../interfaces/components.interface'
+import { BaseTask, TaskType } from '../generated-client/exam'
+import { ModalPropsCreate } from '../interfaces/components.interface'
+import NotificationContext from '../NotificationContext'
+
+
+export default function TaskCreateModal(props: ModalPropsCreate) {
+  const [, showNotification] = React.useContext(NotificationContext)
 
-export default function TaskTemplateCreateModal(props: ModalComponentProps<TaskOut>) {
   const [task, setTask] = React.useState<BaseTask>({
-    workflow_id: undefined,
+    workflow_id: props.parentId,              // eslint-disable-line camelcase
+    name: '',
     description: '',
     type: TaskType.ProcessingTask,
-    status: {},
+    status: 'PENDING',
     args: {},
     artifacts: {},
     destinations: {},
-    is_template: true,
-    is_frozen: false,
+    is_template: props.createTemplate,        // eslint-disable-line camelcase
```

Comment:
I think we only want instances to be created from templates. This is why I would keep the template flag hard coded here.

Reply:
See my comment above, I would like to keep it.